src/semantic-ui/site/collections/form.overrides

```diff
@@ -27,7 +27,6 @@ Form Overrides - REACT-INVENIO-APP-ILS
     position: absolute;
     right: -0.4em;
     top: -0.3em;
-    z-index: 9999;
```

Comment:
did you check if something else didn't get broken because of removing it?

Reply:
I checked most of the Document form elements like `urls`, `internal_notes` and `keywords`. I have checked out also `locations` form and `Series`. After removing it it also plays well with the calendars apart from the dropdowns. I believe is a leftover from the first implementation.